app/interfaces/api/v1/claim_params_helper.rb

```diff
@@ -6,7 +6,7 @@ module ClaimParamsHelper
     #
     params :common_params do
       optional :api_key, type: String, desc: 'REQUIRED: The API authentication key of the provider'
-      optional :creator_email, type: String, desc: 'REQUIRED: The ADP administrator account email address that uniquely identifies the creator of the claim.'
+      optional :creator_email, type: String, desc: I18n.t('api.v1.common_params.creator_email')
```

Comment:
This is the kind do of thing where we have a choice between doing it piecemeal, just to satisfy rubocop, or "all or nothing". Will leave it to your judgment

Reply:
Piecemeal FTW! :) Also, added an issue to address this... at some point!